### PR TITLE
1.0.0 rc1 release process

### PR DESCRIPTION
**Pull request by bitcartel**

### CODE REVIEW

File: contrib/DEBIAN/changelog [hunk 1]
```diff
@@ -1,5 +1,5 @@
-zcash (1.0.0-rc1) UNRELEASED; urgency=medium
+zcash (1.0.0-rc1) jessie; urgency=medium
 
-  * Initial packaging for Debian.
+  * Initial packaging for Debian.
```
Comment (str4d) [inline contrib/DEBIAN/changelog]: Trailing space.

**Comment by ebfull** [comment]
ACK, I don't understand the `25` thing though.

**Comment by str4d** [comment]
@ebfull it is explained in `doc/release-process.md`.

**Comment by bitcartel** [comment]
@zkbot r+

**Comment by daira** [comment]
Post-hoc ACK

**Comment by ageis** [comment]
Also post-hoc ACK

**Pull request merged**